src/actions/arcActions.ts

```diff
@@ -446,25 +446,24 @@ async function getRedemptions(avatarAddress: string, votingMachineInstance: Arc.
   };
 
   // Beneficiary rewards
-  if (proposal.beneficiaryAddress == accountAddress) {
-    if (proposal.state == ProposalStates.Boosted && proposal.winningVote === VoteOptions.Yes) {
-      // Boosted proposal that passed by expiring with more yes votes than no
-      //   have to manually calculate beneficiary rewards
-
-      const votableProposals = await (await proposalInstance.getVotableProposals(avatarAddress))({proposalId}, {fromBlock: 0}).get();
-      const executedProposals = await (await proposalInstance.getExecutedProposals(avatarAddress))({proposalId}, {fromBlock: 0}).get();
-      const proposals = [...votableProposals, ...executedProposals];
-      const numberOfPeriods = proposals[0].numberOfPeriods;
-
-      redemptions.beneficiaryEth = numberOfPeriods * proposal.ethReward;
-      redemptions.beneficiaryNativeToken = numberOfPeriods * proposal.nativeTokenReward;
-      redemptions.beneficiaryReputation = numberOfPeriods * proposal.reputationChange;
-    } else {
-      redemptions.beneficiaryEth = (await proposalInstance.contract.getPeriodsToPay(proposalId, avatarAddress, ContributionRewardType.Eth)) * proposal.ethReward;
-      redemptions.beneficiaryNativeToken = (await proposalInstance.contract.getPeriodsToPay(proposalId, avatarAddress, ContributionRewardType.NativeToken)) * proposal.nativeTokenReward;
-      redemptions.beneficiaryReputation = (await proposalInstance.contract.getPeriodsToPay(proposalId, avatarAddress, ContributionRewardType.Reputation)) * proposal.reputationChange;
-    }
+  if (proposal.state == ProposalStates.Boosted && proposal.winningVote === VoteOptions.Yes) {
```

Comment:
I think this is going to be confusing. how do we track if the beneficiary rewards are for the current user or for someone else? actually this will be easier with my new architecture coming right up